docs/isl-2-0/spec.md

```diff
@@ -0,0 +1,673 @@
+
+{% comment %}{% include example.md title="A schema with one type and no header or footer" code_file="examples/placeholder.isl" %}{% endcomment %}
+
+## Version Marker
+
+{% include grammar-element.md productions="isl_version_marker" %}
+
+The Ion Schema version marker for Ion Schema 2.0 is `$ion_schema_2_0`.
+The Ion Schema version marker must appear in the schema document before any `type` or `schema_header` structs.
+If a schema document has no version marker, then it is an Ion Schema 1.0 document, and it must follow the [Ion Schema 1.0 specification](../isl-1-0/spec).
+If any Ion Schema version marker is found after the first `schema_header` or `type`, the schema document is invalid, and it will result in an error.
+
+{% comment %}{% include example.md title="An ISL 2.0 Schema" code_file="examples/placeholder.isl" %}{% endcomment %}
+
+## Imports
+
+{% include grammar-element.md productions="imports_declaration,import,import_schema,import_type,import_type_alias" %}
+
+An import allows types from other schemas to be used within a schema definition.
+An import that only specifies an `id` makes all the types from that schema available for use in the current schema.
+Specifying a `type` narrows the import to that single type, and a type may be imported with a different name by specifying: `as: <TYPE_ALIAS>`.
+The core types and Ion types are implicitly imported before any specified imports; specified imports are performed in order.
+Once an implementation has finished loading a schema, if any imports could not be resolved, it must result in an error. 
+If two types with the same name are imported, or if a type defined within a schema has the same name as an imported type, this must result in an error.
+Only named types of a schema may be imported in another schema.
+Types may only be imported from the schema where they are declared; importing a type to a schema does not make that type transitively available to any other schemas. 
+
+{% comment %}{% include example.md title="A schema that imports types from other schemas" code_file="examples/placeholder.isl" %}{% endcomment %}
+
+### Schema Authorities
+
+The structure of a `id` string is defined by the schema authority responsible for the schema/type(s) being imported.
+Note that runtime resolution of a schema over a network presents availability and security risks, and should therefore be avoided.
+
+When resolving a schema, authorities may choose to follow well-known patterns; for example:
+- a filesystem authority might specify that an `id` string corresponds to an ISL file relative to some base, e.g.:`"{base}/com/example/core/Customer.isl"`
+- a REST authority might specify that an `id` string is a resource URL that corresponds to an ISL file, e.g.: `"https://{host}:{port}/{base}/com/example/core/Customer"` (again, note the inherent availability and security risks here)
+
+Beware that the choice of authority affects the way schemas can be imported by other schemas.
```

Comment:
I chose to just remove the statement since I couldn't think of any good clarifying example. I don't think there actually is anything inherently dangerous with choosing the "wrong" authority. You just need to make sure that (as mentioned a few lines earlier):

>  structure of an `id` string is defined by the schema authority responsible for the schema/type(s) being imported

... and so if you have an `id` string (in an import), you need to make sure that you have the authority that is responsible for that schema id.